lib/api/github.js

```diff
@@ -223,17 +224,7 @@ async function getFileContent(filePath, branchName = config.baseBranch) {
 }
 
 async function getFileJson(filePath, branchName = config.baseBranch) {
-  try {
-    const file = await getFile(filePath, branchName);
-    return JSON.parse(new Buffer(file, 'base64').toString());
-  } catch (error) {
-    if (error.statusCode === 404) {
-      // If file not found, then return null JSON
-      return null;
-    }
-    // Propagate if it's any other error
-    throw error;
-  }
+  return JSON.parse(await getFileContent(filePath, branchName));
```

Comment:
Since there's nothing GitHub specific, this function could probably be removed, or moved into a helper file. Same with the `getFileContent` function. A little bit less duplication between GitHub and GitLab API files.